### PR TITLE
Burn - Simple Calorie Counter

### PR DESCRIPTION
**Pull request by slynchDev**
Calorie count persists to kcal.txt and clears at midnight. Increment (step) value can be changed as needed

### CODE REVIEW

File: apps/burn/burn.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+let drag, y, x;
+
+// If the hardware version is Bangle.js 2, set the coordinates accordingly
+if (BANGLEJS2) {
+  drag = undefined;
+  y = 45;
+  x = 5;
+} else {
+  // If the hardware version is Bangle.js 1, set the coordinates accordingly
+  y = 100;
```
Comment (thyttan) [inline apps/burn/burn.js:151]: Lint warning: `'y' is assigned a value but never used`

So seems like you could just remove it?

File: apps/burn/burn.js [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  x = 25;
```
Comment (thyttan) [inline apps/burn/burn.js:152]: And likewise here: `'x' is assigned a value but never used`

**Review by thyttan — CHANGES_REQUESTED** [review]

File: apps/burn/.prettierignore
Comment (thyttan) [inline apps/burn/.prettierignore:1]: I think this is just for formatting locally on your machine? We'll probably want to not have that file on the repo. What do you think @bobrippling @halemmerich ?

**Comment by thyttan** [comment]
Thanks! I left some comments. 

If you want to let the calculation run in the background you could implement it in a `burn.boot.js` file or as a widget. But no need to do that in this PR - just a feature suggestion for the future.

Also I'd prefer if filenames were renamed from 
`burn.js`, `burn_icon.png` and `burn-icon.js`
 to simply 
`app.js`, `app.png` and `app-icon.js` as is common in the repo.

**Comment by slynchDev** [comment]
thank you, I'll make your suggested changes

**Comment by thyttan** [comment]
OK - looking forward to see it when your ready again! 🙂

File: apps/burn/.prettierignore
Comment (bobrippling) [reply apps/burn/.prettierignore:1]: Yeah I agree, thanks for taking this :)

**Pull request closed without merging**